lib/cli.js

```diff
@@ -47,6 +48,30 @@ const mkdir = promisify(fs.mkdir);
 const stat = promisify(fs.stat);
 const writeFile = promisify(fs.writeFile);
 
+/**
+ * Loads plugins with the specified names.
+ * @param {{ "import": (name: string) => Promise<any> }} importer An object with an `import` method called once for each plugin.
+ * @param {string[]} pluginNames The names of the plugins to be loaded, with or without the "eslint-plugin-" prefix.
+ * @returns {Promise<Record<string, Plugin>>} A mapping of plugin short names to implementations.
+ */
+async function loadPlugins(importer, pluginNames) {
+    const plugins = {};
+
+    await Promise.all(pluginNames.map(async pluginName => {
+
+        const shortName = naming.getShorthandName(pluginName, "eslint-plugin");
+        const longName = naming.normalizePackageName(pluginName, "eslint-plugin");
+        const module = await importer.import(longName);
+
+        if (!("default" in module)) {
+            throw Error(`"${longName}" cannot be imported because it does not provide a default export`);
```

Comment:
[Question] Elsewhere outside of `.md` files and `tests/`, it's always `throw new Error` - not `throw Error`. I.e. [`no-throw-literal`](https://eslint.org/docs/latest/rules/no-throw-literal) has been adhered to. Is it intentional that this doesn't?

```suggestion
            throw new Error(`"${longName}" cannot be imported because it does not provide a default export`);
```

Reply:
I think it's just a matter of style. `throw Error()` is the same as `throw new Error()`. `no-throw-literal` does not flag it ([playground](https://eslint.org/play/#eyJ0ZXh0IjoiLyogZXNsaW50IG5vLXRocm93LWxpdGVyYWw6IFwiZXJyb3JcIiAqL1xuXG50aHJvdyBFcnJvcihcImZvb1wiKTsiLCJvcHRpb25zIjp7ImVudiI6e30sInJ1bGVzIjp7fSwicGFyc2VyT3B0aW9ucyI6eyJlY21hRmVhdHVyZXMiOnt9LCJlY21hVmVyc2lvbiI6ImxhdGVzdCIsInNvdXJjZVR5cGUiOiJtb2R1bGUifX19)). In my repos, I have a rule that reports unnecessary `new` expressions, so I have a tendency to write just `Error()`. `eslint-plugin-unicorn` has [this rule](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/throw-new-error.md) that requires `new Error` in throw statements.